docker-compose.example.yml

```
@@ -0,0 +1,12 @@
+version: '3.1'
+services:
+  reactmap:
+    image: ghcr.io/WatWowMap/reactmap:main #pr-10
+    container_name: reactmap
+    command: sh -c "sudo yarn start"
```

```suggestion
    command: "yarn start"
```
Don't need (and can't use `sudo` with alpine)